src/NzbDrone.Core.Test/ThingiProviderTests/ProviderStatusServiceFixture.cs

```diff
@@ -161,7 +161,7 @@ public void should_not_escalate_further_till_after_5_minutes_since_startup()
             status.Should().NotBeNull();
 
             origStatus.EscalationLevel.Should().Be(3);
-            status.DisabledTill.Should().BeCloseTo(_epoch + TimeSpan.FromMinutes(5), _disabledTillPrecision);
+            status.DisabledTill.Should().BeCloseTo(_epoch + TimeSpan.FromMinutes(1), _disabledTillPrecision);
```

Comment:
During the start up grade period (15 minutes), do we want things to retry every minute? Has the current default of 5 been an issue?

Reply:
If Prowlarr is reporting to back off via 429 responses I'd expect Sonarr to respect those, is it not doing that?